### ISSUE TITLE
[BUG] Game banners in reviews link to discussion page

### ISSUE THREAD

**Issue by kazerniel** [IsThereAnyDeal/AugmentedSteam#2040]
### Expected Behavior
I think they used to link to the main community page, not discussions. Though tbh I'd prefer if they took me to the store page like in the activity feed 😅

### Actual Behavior
The banner links to the discussions boards of the game.

### Steps to Reproduce the Problem

  1. Go to any review
  2. Click on banner

### Specifications

  - Augmented Steam Version: 3.1.1
  - Browser Version: Firefox 129.0.2

**Comment by candela97** [comment]
You have probably configured "Default community home page tab" to "Discussions", but there should a be link below that links to the store page?

![螢幕擷取畫面 2024-09-01 182838](https://github.com/user-attachments/assets/c8d720e9-05ba-4f83-ba00-6d75b3311598)

**Comment by kazerniel** [comment]
Sorry, I wrote the ticket late at night, I meant the banners in the list of someone's reviews:

![image](https://github.com/user-attachments/assets/f357f74d-7722-4d67-9d32-6f89b49703a8)

Thanks, I forgot that there was even a setting like that - when I set it up in the past I didn't think it would affect links aside of the "Community Hub" link on the top of store pages. While on the product pages it's more helpful if it goes straight to discussions, when reading someone's reviews I'd prefer to be able to see the games they are about without having to click through to each review.

I guess I haven't noticed this AS behaviour before because I typically encounter these banners on my activity feed for which there's already a setting to go to the store page. So could we turn this ticket into a feature request for a similar checkbox for banners on lists of reviews? 😅

**Comment by candela97** [comment]
> I guess I haven't noticed this AS behaviour before because I typically encounter these banners on my activity feed for which there's already a setting to go to the store page. So could we turn this ticket into a feature request for a similar checkbox for banners on lists of reviews? 😅

Yeah that setting can be expanded to cover more pages, but a reminder to myself that it'll have to be re-applied after sorting reviews.
